src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "./utils/variabels";
+@import "./blocks/reset.scss";
+@import "./blocks/content.scss";
+@import "./blocks/calender.scss";
```

Comment:
Add new line
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file